.travis.yml

```diff
@@ -10,11 +10,14 @@ env:
     - TRIGGER_REPO_BRANCH: "master"
 
 before_install:
-  - 'git checkout -B $TRAVIS_BRANCH' # Reconcile detached HEAD
-  - 'npm install -g bower grunt-cli'
+  - if [[ `npm -v` != 3* ]]; then npm i -g npm@3; fi
+  - npm install -g bower grunt-cli
```

Comment:
We don't need bower to build, but Travis still needs to verify the bower install during the release process. In order for the Travis build to succeed, we still need this line -- for now.

FYI, the Travis build (i.e., the scripts from patternfly-eng-release) will still perform an npm and bower install for all our repos. That is, if there is a bower.json file available. During a release, the scripts will also perform an npm and bower install to verify.